SpringProject/src/it/java/com/gradproject2019/ConferenceControllerIT.java

```diff
@@ -150,6 +150,32 @@ public void shouldReturn400WhenConferenceInPast() throws URISyntaxException {
         Assert.assertEquals(400,responseString.getStatusCodeValue());
     }
 
+    @Test
+    public void shouldReturn404WhenConferenceToBeDeletedNotFound() throws URISyntaxException {
+        //given
+        URI uri = new URI(baseUrl + "/1000000000");
+
+        //when
+        ResponseEntity<String> responseString = deleteConference(uri);
+
+        //then
+        Assert.assertEquals(404,responseString.getStatusCodeValue());
+    }
+
+    @Test
+    public void shouldReturn200AndDeleteConference() throws URISyntaxException {
+        //given
+        Conference addedConference = conferenceRepository.saveAndFlush(conference);
+        URI uri = new URI(baseUrl + "/" + addedConference.getId());
+
+        //when
+        ResponseEntity<String> responseString = deleteConference(uri);
```

Comment:
Don't need String in the name here, we can see that from the object type. response is fine